alpha/apps/kaltura/lib/kObjectCreatedHandler.php

```diff
@@ -80,4 +84,20 @@ public function syncEntryEntitlementInfo(entry $vodEntry, LiveEntry $liveEntry)
 			
 		$vodEntry->save();
 	}
+	
+	public function syncLiveEntryThumbnail (entry $object, LiveEntry $liveEntry)
+	{
+		//Get live entry thumbnails
+		$criteria = new Criteria();
+		$criteria->add (assetPeer::ENTRY_ID, $liveEntry->getId());
+		$criteria->add(assetPeer::STATUS, asset::ASSET_STATUS_READY);
+		$criteria->add(assetPeer::TYPE, assetType::THUMBNAIL);
+		$thumbAssetList = assetPeer::doSelect($criteria);
```

Comment:
Please use `assetPeer::retrieveThumbnailsByEntryId` or `assetPeer::retrieveByEntryId`.
Anyway, you probably would like also types the extend thumbnail, so use: `KalturaPluginManager::getExtendedTypes(self::OM_CLASS, assetType::THUMBNAIL)`.